Dockerfile-linux.template

```
@@ -307,10 +307,23 @@ RUN set -eux; \
 		"pip==$PYTHON_PIP_VERSION" \
 {{ if .setuptools then ( -}}
 		"setuptools==$PYTHON_SETUPTOOLS_VERSION" \
```

For completeness, and in case `get-pip` were to ever stop installing `wheel` by default for older Python versions, should we add `wheel` as an explicit dependency here (without a version specfier, to match the `get-pip` behaviour)?

ie:
```suggestion
		"setuptools==$PYTHON_SETUPTOOLS_VERSION" \
		wheel \
```

---

Ah yeah, that's probably a good idea, even though I don't love it :joy: :sob: :heart: